src/test/java/rx/plugins/RxJavaPluginsTest.java

```diff
@@ -108,6 +108,14 @@ public void testObservableExecutionHookViaRegisterMethod() {
     }
 
     @Test
+    public void testSingleExecutionHookViaRegisterMethod() {
```

Comment:
Majority of test methods which we have are prefixed by `test`. [Contribution](https://github.com/ReactiveX/RxJava/blob/1.x/CONTRIBUTING.md) doc states to follow existing conventions. Do we have some separated guidelines for tests convention somewhere?